wakuv2/waku.go

```diff
@@ -1800,7 +1800,7 @@ func (w *Waku) StopDiscV5() error {
 }
 
 func (w *Waku) ConnectionChanged(state connection.State) {
-	if !state.Offline && !w.onlineChecker.IsOnline() {
+	if !w.cfg.LightClient && !state.Offline && !w.onlineChecker.IsOnline() {
```

Comment:
Because the underlying channel is not being consumed in case of lightClient, did not make sense to try to push into it. I noticed the log being printed in line 1807 in case of lightClients which is un-necessary.

Notice that lightClient related connectionChange is being handled as of now at https://github.com/status-im/status-go/blob/fadecf42781f1b062dd207363eb327f169d6b6f3/wakuv2/waku.go#L1370-L1382